branchpro/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""branchpro is a Branching Processes modelling library.
+It contains functionality for modelling, simulating, and visualising the
+number of cases of infections by day during an outbreak of the influenza virus.
+"""
+
+# Import version info
+from .version_info import VERSION_INT, VERSION  # noqa
+
+# Import main classes
+from .models import ForwardModel, BranchProModel    # noqa
+from .simulation import SimulationController    # noqa
+from .visualisation import Visualisation     # noqa
```

Comment:
There's a warning about no newline. I think it's referring to [this requirement](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

Reply:
Noted; made the required change